table/model.go

```diff
@@ -72,3 +82,69 @@ func New(columns []Column) Model {
 func (m Model) Init() tea.Cmd {
 	return nil
 }
+
+func (m Model) isRowMatched(row Row) bool {
+	for _, column := range m.columns {
+		if column.Filterable {
+			data, ok := row.Data[column.Key]
+			if !ok {
+				continue
+			}
+			if strings.Contains(strings.ToLower(data.(string)), strings.ToLower(m.filterTextInput.Value())) {
+				return true
+			}
+		}
+	}
+	return false
+}
+
+func (m Model) getFilteredRows(rows []Row) []Row {
+	if !m.filtered || m.filterTextInput.Value() == "" {
+		return rows
+	}
+
+	filteredRows := make([]Row, 0)
+
+	for _, row := range rows {
+		if m.isRowMatched(row) {
+			filteredRows = append(filteredRows, row)
+		}
+	}
+
+	return filteredRows
+}
+
+// GetRows return sorted and filtered rows
+func (m Model) GetRows() []Row {
```

Comment:
I think we should rename this to `GetVisibleRows`, because `GetRows` implies it returns all the rows.

Reply:
`GetVisibleRows` seems implies the rows that this page will shown. emm~ but it is better then `GetRows`.

Reply:
That's a very good point.  It doesn't take paging into account, but `GetVisibleRows` sounds like it should.  Hmm.

I'm open to ideas on this one, I'm not quite sure what to call it.  `GetSortedAndFilteredRows` feels a little silly, but... maybe?

Reply:
I still using `GetVisibleRows` now.

Maybe `rows` for visible rows, and `allRows` for original rows. If that solution is approved, it can be another Pull Request.